packages/cozy-harvest-lib/test/__mocks__/cozy-keys-lib/index.js

```diff
@@ -18,6 +18,7 @@ const UriMatchType = {
   Domain: 'Domain'
 }
 
+
```

Comment:
Looks like this "wip" commit can be removed (https://github.com/cozy/cozy-libs/pull/956/commits/2103878394043bf5ce661167bdfa08391c37d32f)